47.md

```diff
@@ -373,6 +373,33 @@ Response:
 }
 ```
 
+### `get_budget`
+
+Budgets can be used to limit the amount of funds that can be spent by a connection in a given time period.
+Budgets are optional and can be set by the user if their wallet supports it. Support for this command indicates
+that the wallet supports user-configured budgets. The budget will be reset at the `renews_at` timestamp.
+
+Request:
+```jsonc
+{
+    "method": "get_budget",
+    "params": {
+    }
+}
+```
+
+Response:
+```jsonc
+{
+    "result_type": "get_budget",
+    "result": {
+        "remaining_budget_msats": 10000,
+        "total_budget_msats": 100000,
+        "renews_at": 1693876973, // timestamp in seconds since epoch
```

Comment:
I think it would be good to include the renewal period too because normally in a UI you'd see the renewal period if it shows budgeting info. Alby Hub's supported renewal periods are: `"daily" | "weekly" | "monthly" | "yearly" | "never"`.

Reply:
Good callout, that's probably worthwhile. Added